docs/UserGuide.adoc

```diff
@@ -70,12 +70,47 @@ These are just a few of the commands you can execute on EatMe. Refer to <<Featur
 
 Format: `help`
 
+==== Listing all eateries: `list`
+
+Shows a list of all eateries in the app.  +
+Format: `list`
+
+==== Clearing all entries: `clear`
+
+Format: `clear`
+
+==== Importing eateries: `import`
+
+*Coming in v1.4*
+
+Imports eateries from a text file. +
+Format: `import \f [filename]`
+
+Examples:
+
+* `import \f /home/user/eatme-backup-john.txt`
+
+==== Exporting eateries: `export`
+
+*Coming in v1.4*
+
+Exports eateries to a text file. +
+Format: `export \f [filename]`
+
+Examples:
+
+* `export \f /home/user/eatme-backup-john.txt`
+
 ==== Exit the app: `exit`
 
 Format: `exit`
 
-=== Working with eateries
-==== Adding an eatery: `add`
+=== Main Mode
+
+In Main Mode, manipulation of eateries which user had been to will be available. +
```

Comment:
In Main mode, users will be able to manipulate eateries which users have visited.